R/RSF/build_tarballs.jl

```diff
@@ -0,0 +1,1888 @@
+    ExecutableProduct("sfconjgradmpi", :sfconjgradmpi),
+    ExecutableProduct("sfnorm2", :sfnorm2),
+    ExecutableProduct("sfcameron2d", :sfcameron2d),
+    ExecutableProduct("sfricker1", :sfricker1),
+    ExecutableProduct("sftest1_matchl1", :sftest1_matchl1),
+    ExecutableProduct("sfautocorr", :sfautocorr),
+    ExecutableProduct("sfdivn2d", :sfdivn2d),
+    ExecutableProduct("sflopef", :sflopef),
+    ExecutableProduct("sfadjtest1", :sfadjtest1),
+    ExecutableProduct("sfpsti2d", :sfpsti2d),
+    ExecutableProduct("sfpde2dadp", :sfpde2dadp),
+    ExecutableProduct("sfdipfilter", :sfdipfilter),
+    ExecutableProduct("sffocus", :sffocus),
+    ExecutableProduct("sfmodrefl3", :sfmodrefl3),
+    ExecutableProduct("sfcompensate", :sfcompensate),
+    ExecutableProduct("sfmpiewertm", :sfmpiewertm),
+    ExecutableProduct("sfmoveout", :sfmoveout),
+    ExecutableProduct("sfopame2dckzz", :sfopame2dckzz),
+    ExecutableProduct("sfsmoothn", :sfsmoothn),
+    ExecutableProduct("sfwlslfdc1tw2", :sfwlslfdc1tw2),
+    ExecutableProduct("sfchebyshevp", :sfchebyshevp),
+    ExecutableProduct("sfsemblancew", :sfsemblancew),
+    ExecutableProduct("sffindwellcoord", :sffindwellcoord),
+    ExecutableProduct("sftahagc", :sftahagc),
+    ExecutableProduct("sftahremoveclick", :sftahremoveclick),
+    ExecutableProduct("sfgetcretimecurve", :sfgetcretimecurve),
+    ExecutableProduct("sfgpi3dzo", :sfgpi3dzo),
+    ExecutableProduct("sfewelr3dgrad", :sfewelr3dgrad),
+    ExecutableProduct("sfsplinefilter", :sfsplinefilter),
+    ExecutableProduct("sfupsamp1", :sfupsamp1),
+    ExecutableProduct("sfinvtest1", :sfinvtest1),
+    ExecutableProduct("sftlagtoang2d", :sftlagtoang2d),
+    ExecutableProduct("sfpsmig", :sfpsmig),
+    ExecutableProduct("sfcell2", :sfcell2),
+    ExecutableProduct("sfinterp2", :sfinterp2),
+    ExecutableProduct("sfheat3", :sfheat3),
+    ExecutableProduct("sfmarchenko", :sfmarchenko),
+    ExecutableProduct("sfmpisglrrtm2", :sfmpisglrrtm2),
+    ExecutableProduct("sfvoft", :sfvoft),
+    ExecutableProduct("sfarrival", :sfarrival),
+    ExecutableProduct("vpwhitepruf", :vpwhitepruf),
+    ExecutableProduct("sfdeburst", :sfdeburst),
+    ExecutableProduct("sfsharpen", :sfsharpen),
+    ExecutableProduct("sfosam3dckzz", :sfosam3dckzz)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="SuiteSparse_jll", uuid="bea87d4a-7f5b-5778-9afe-8cc45184846c"))
+    Dependency(PackageSpec(name="FFMPEG_jll", uuid="b22a6f82-2f65-5046-a5b2-351ab43fb4e5"))
+    Dependency(PackageSpec(name="Cairo_jll", uuid="83423d85-b0ee-5818-9007-b63ccbeb887a"))
+    Dependency(PackageSpec(name="MPICH_jll", uuid="7cb0a576-ebde-5e09-9194-50597f1243b4"))
+    Dependency(PackageSpec(name="MicrosoftMPI_jll", uuid="9237b28f-5490-5468-be7b-bb81f5f5e6cf"))
```

Comment:
For MPI please look at https://github.com/JuliaPackaging/Yggdrasil/blob/3f318236dead5863a703287db226946b62b83794/P/P4est/build_tarballs.jl#L66-L70 https://github.com/JuliaPackaging/Yggdrasil/blob/3f318236dead5863a703287db226946b62b83794/P/P4est/build_tarballs.jl#L78